docs/advanced-recipes/large-objects.mdx

```diff
@@ -104,8 +104,9 @@ Then we use it in our component:
 import { atom, useAtom } from 'jotai'
 import { selectAtom, splitAtom } from 'jotai/utils'
 
-const Tags: React.FC = () => {
-  const tagsAtom = selectAtom(readOnlyInfoAtom, (s) => s.tags)
+const selector = (s) => s.tags
```

Comment:
```suggestion
const selectTags = (s) => s.tags
```
How's this?

Reply:
@dai-shi Yeah or tagsSelector, both ok to me